Make sure the daemon script isn't running before doing
any of this stuff.

If you just pull'ed, it's probably a good idea to rebuild
the apps.  Use the export script for that:

  python export_script.py

Then start up the daemon:

  python daemon.py
